protos/exocore/core/config.proto

```diff
@@ -33,6 +35,34 @@ message NodeCellConfig {
     }
 }
 
+message NodeStoreConfig {
+    EntityIndexConfig index = 1;
+}
+
+/// Configuration of the entities index
+message EntityIndexConfig {
+    uint64 chain_index_min_depth = 1;
```

Comment:
Would probably leave some comments here on what the fields are about. Just copy paste what was in rust already?